plugin-build/gradle.properties

```diff
@@ -0,0 +1,8 @@
+ID=io.sentry.kotlin.multiplatform.gradle.plugin
+VERSION=1.0.0
```

Comment:
i see this is still from the template, I guess you're gonna change, just don't forget to align the version with the SDK version itself as we've agreed